docs/installation/installation.md

```diff
@@ -7,12 +7,12 @@ title: Installation Overview
 ### Karmada kubectl plugin
```

Comment:
Here both `kubectl-karmada` and `karmadactl` are present in `Prerequistites`, do you mean people need to install them both?

Reply:
Not yet. I think people install one of them is ok. Add it to the note.